File: py/level2.py
def answer(n):
    st = ''
    for i in range(2, 10000):
        j = 2
        while j <= i:
            if i == j:
                st += str(i)
            if (i % j) == 0:
                break
            j += 1

    return st[n:n+5]

File: js/level2.js
let i, j;
let rs = '';
for (i = 2; i < 30000; i++) {
    console.log(j)
    for (j = 2; j <= i; j++) {
        if (i == j) {
            rs = rs + i;
        }
        if (i % j == 0) {
            break;
        }
    }
}

function answer(n) {
    return rs.split('').slice(n, n + 5).join('');
}

// console.log(rs);